examples/src/components/AttributeFilterExample.jsx

```diff
@@ -1,26 +1,114 @@
+    onLoadingChanged(...params) {
         // eslint-disable-next-line no-console
-        console.log("AttributeFilterExample onApply", ...params);
+        console.info("AttributeFilterExample onLoadingChanged", ...params);
+    }
+
+    onApply(filter) {
+        // eslint-disable-next-line no-console
+        console.log("AttributeFilterExample onApply", filter);
+        const isPositive = !!filter.in;
+        const elementsProp = isPositive ? "in" : "notIn";
+        this.setState({ error: null });
+        if (filter[elementsProp].length !== 0) {
+            const filters = [
+                {
+                    [isPositive ? "positiveAttributeFilter" : "negativeAttributeFilter"]: {
+                        displayForm: {
+                            identifier: filter.id,
+                        },
+                        [elementsProp]: filter[elementsProp].map(
+                            element => `${locationResortUri}/elements?id=${element}`,
+                        ),
+                    },
+                },
+            ];
+            this.setState({ filters });
+        } else if (isPositive) {
+            this.setState({
+                error: "Filter has to select at least an item",
+            });
+        }
+    }
+
+    onError(...params) {
+        // eslint-disable-next-line no-console
+        console.info("AttributeFilterExample onLoadingChanged", ...params);
     }
 
     render() {
+        const { filters, error } = this.state;
+        const afm = {
+            measures: [
+                {
+                    localIdentifier: "totalSales",
+                    definition: {
+                        measure: {
+                            item: {
+                                identifier: totalSalesIdentifier,
+                            },
+                        },
+                    },
+                    alias: "$ Total Sales",
+                    format: "#,##0",
+                },
+            ],
+            attributes: [
+                {
+                    displayForm: {
+                        identifier: locationResortIdentifier,
+                    },
+                    localIdentifier: "locationResort",
+                },
+            ],
+            filters,
+        };
+
         return (
-            <div>
+            <div className="s-attribute-filter">
                 <AttributeFilter
-                    identifier={employeeNameIdentifier}
+                    identifier={locationResortIdentifier}
                     projectId={projectId}
                     fullscreenOnMobile={false}
                     onApply={this.onApply}
                 />
+                <div style={{ height: 300 }} className="s-line-chart">
+                    {error ? (
+                        <ErrorComponent message={error} />
+                    ) : (
+                        <LineChart
+                            projectId={projectId}
+                            afm={afm}
```

Comment:
Is this really correct? Imho AFM components are deprecated for the customers and core components that use bucket interface should be used instead.

```
<LineChart
  projectId={projectId}
  measures={[..]}
  attributes={[...]}
  filters={[...]}
  onLoadingChanged={this.onLoadingChanged}
  onError={this.onError}
/>
```